src/adts/bit_array_impl.h

```diff
@@ -12,7 +12,7 @@
 #include <libpq/pqformat.h>
 
 #include "adts/uint64_vec.h"
-#include "compat.h"
+#include "compat/compat.h"
```

Comment:
This file is now more of a "list of handy things". If we have a subdirectory, it might be good to split this into smaller pieces and add some structure.